source/MapDetailPanel.cpp

```diff
@@ -537,6 +537,8 @@ void MapDetailPanel::DrawKey()
 	font.Draw(header, pos + headerOff, medium);
 	pos.Y() += 20.;
 
+	int numCommodities = 4 + !hasUnexplored + !hasUninhabited;
```

Comment:
Maybe something like `numGovernmentKey` would better describe what this var represents?

Reply:
`openLines` or `availableSpaces` maybe?

Reply:
availableLines sounds good